desktop/core/src/desktop/auth/backend.py

```diff
@@ -878,6 +878,7 @@ def create_user(self, claims):
         email=email,
         first_name=first_name,
         last_name=last_name,
+        groups=ensure_has_a_group(username),
```

Comment:
function ensure_has_a_group is expecting user object not a username string. I think normal this function is calling from auth backend instead.

Reply:
+1 for @wing2fly's comment. `ensure_has_a_group` needs a User object.

I think you can try like this:

```python
user = self.UserModel.objects.create_user(
    username=username,
    email=email,
    first_name=first_name,
    last_name=last_name,
    is_superuser=self.is_hue_superuser(claims)
)

ensure_has_a_group(user)
return user
```

Reply:
addressed it